inc/templates/namespace.php

```diff
@@ -21,6 +21,7 @@ function bootstrap() {
 	add_action( 'wp_irving_component_children', __NAMESPACE__ . '\\inject_favicon', 10, 3 );
 	add_action( 'wp_irving_component_children', __NAMESPACE__ . '\\inject_custom_css', 10, 3 );
 	add_action( 'wp_irving_component_children', __NAMESPACE__ . '\\inject_block_library_styles', 10, 3 );
+	add_action( 'wp_irving_component_children', __NAMESPACE__ . '\\inject_body_classes', 10, 3 );
```

Comment:
Why not handle this directly inside the children callback for the irving/head component?

Reply:
Following the existing pattern, I'm fine with a refactor whenever. Could make an argument that having a filter gives devs an opportunity to unfilter it, but I don't feel super strongly about it.